### PR TITLE
Add an Engineering Allocation view

### LINKED ISSUE #270: Add an Engineering Allocation view
Add a view that includes an individual BugList for each OMC engineer, tracking all unresolved bugs in our components that are assigned to that engineer and have an iteration whose first part (e.g. the 117 in 117.1) matches the current iteration. It should look kind of like the table below. Results should be sorted by points first, then by priority. In other words, 7-pointers go before 3-pointers, but if there are two 7-pointers, then the P1 should go before the P2.

### shughes
| \[check all\] | 3 bugs \| 13 points |              |     |                |           |          |        |
| ------------- | ------------------------------ | ------------ | :-: | -------------- | --------- | -------- | ------ |
|               | Ticket                         | Bug          |     | Title          | Iteration | Priority | Points |
| [ ]           | [OMC-123]()                    | [1234 567]() | ➕  | Lorem ipsum    | 118.1     | P2       | 7      |
| [ ]           | [OMC-456]()                    | [2345 678]() | 📋  | dolor sit amet | 118.1     | P1       | 3      |
| [ ]           | [OMC-789]()                    | [3456 789]() |  ☀  | consectetur    | 118.1     | P1       | 3      |

### PR DESCRIPTION
**Pull request by aminomancer**
Fixes #270

### CODE REVIEW

**Comment by aminomancer** [comment]
Could add more bells and whistles, but this should work for #270 for now. You can also use this for reference for #271, I think. Similar queries, but just processing the bugs differently.

File: src/content/components/AllocationView/AllocationView.js [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+import { columnTransforms } from "../BugList/columnTransforms";
+
+// Add a view that includes an individual BugList for each OMC engineer,
+// tracking all unresolved bugs in our components that are assigned to that
+// engineer, have an iteration matching the current release, and have the points
+// field set. Display columns should include ticket, bug id, type, title,
+// iteration, priority, and points, in that order. Bugs should be sorted by
+// points first, then by priority. In other words, 7-pointers go before
+// 3-pointers, but if there are two 7-pointers, then the P1 should go before the
+// P2.
```
Comment (aminomancer) [inline src/content/components/AllocationView/AllocationView.js:18]: These longer notes are just for development to make the spec easier to find. I'll remove em when we're done

File: src/content/img/icons/capacity.svg
Comment (aminomancer) [inline src/content/img/icons/capacity.svg:1]: I should really reverse the colors here, bigger is badder so the big square should be red and the small square green. I can do that before we land it

**Review by yoenzhang — APPROVED** [review]
Looks good! Tested on local, everything seems to work as intended

**Comment by aminomancer** [comment]
Awesome, thanks for checking. I'll fix up my comments and merge

**Pull request merged**